From 86cd0ce316287e81090343418cc806b4f3274bbe Mon Sep 17 00:00:00 2001
From: Mathieu Trudel-Lapierre <mathieu.trudel-lapierre@canonical.com>
Date: Thu, 28 Feb 2019 10:07:21 -0500
Subject: Update sourcefiles linked from arm_efi arch to fix symbol checking

Signed-off-by: Mathieu Trudel-Lapierre <mathieu.trudel-lapierre@canonical.com>
---
 grub-core/Makefile.core.def | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/grub-core/Makefile.core.def b/grub-core/Makefile.core.def
index cdeb6bd9b..13600ee11 100644
--- a/grub-core/Makefile.core.def
+++ b/grub-core/Makefile.core.def
@@ -1706,6 +1706,7 @@ module = {
   ia64_efi = loader/ia64/efi/linux.c;
   arm_coreboot = loader/arm/linux.c;
   arm_efi = loader/arm64/linux.c;
+  arm_efi = loader/efi/linux.c;
   arm_uboot = loader/arm/linux.c;
   arm64 = loader/arm64/linux.c;
   arm64 = loader/efi/linux.c;
@@ -1785,6 +1786,7 @@ module = {
 module = {
   name = chain;
   efi = loader/efi/chainloader.c;
+  efi = loader/efi/linux.c;
   i386_pc = loader/i386/pc/chainloader.c;
   i386_coreboot = loader/i386/coreboot/chainloader.c;
   i386_coreboot = lib/LzmaDec.c;
